ios/Mattermost/Info.plist

```diff
@@ -64,25 +64,25 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Share post data across devices with Mattermost</string>
 	<key>NSCalendarsUsageDescription</key>
-	<string>Share contacts in your Mattermost instance</string>
+	<string>Share contacts in your Mattermost workspace</string>
 	<key>NSCameraUsageDescription</key>
-	<string>Take a Photo or Video and upload it to your Mattermost instance</string>
+	<string>Take a Photo or Video and upload it to your Mattermost workspace</string>
```

Comment:
```suggestion
	<string>Take a Photo or Video and upload it to Mattermost</string>
```

Reply:
Question: Is there a specific reason why ``Photo`` and ``Video`` are capitalized here? If not, let's lowercase them, please.